test/turnt.toml

```diff
@@ -54,10 +54,14 @@ command = "odgi matrix -i {filename}"
 binary = true
 command = "python3 ../slow_odgi/matrix.py < {filename}"
 
+[envs.validate_setup]
+binary = true
+command = "python3 ../slow_odgi/perturb.py < {filename} > temp.{filename}"
+
 [envs.validate_oracle]
 binary = true
-command = "odgi validate -i {filename}"
+command = "odgi validate -i temp.{filename} 2>temp.txt; cat temp.txt"
```

Comment:
`odgi validate` puts its output into stderr. What I really want is to direct stderr to stdout, which will itself go to filename.out thanks to turnt. However, this isn't working. I think it has to do with the order, as suggested by [these folks](https://linuxize.com/post/bash-redirect-stderr-stdout/#redirecting-stderr-to-stdout).

My workaround: I'm throwing stderr into a temp file, and then using `cat` to essentially echo it back to turnt, which saves it to filename.out.

Reply:
Aha, that is annoying. It would be good if `2>&1` worked in Turnt commands. TBH I am not exactly sure how to accomplish this, but would you mind opening a Turnt bug and we can go from there?

Reply:
Done!